### PR TITLE
Cross browser fixes

### PR DESCRIPTION
**Pull request by bz-stripe**
### Changes
- [x] use ES5 syntax
- [x] add `fetch` and `promise` polyfills
- [x] fix result caching bug
- [x] fix `return_url` bug for IE11
- [x] fix some CSS lint warnings

### Testing
Tested on IE11 and OSX Chrome

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -141,9 +142,9 @@ app.post('/webhook', async (req, res) => {
     case 'identity.verification_intent.updated':
     case 'identity.verification_intent.succeeded':
       cache.upsert(data.id, data);
-      const { socketId } = getStaticValue(data.id, 'socketId');
+      const socketId = cache.getStaticValue(data.id, 'socketId');
       if (socketId) {
-        io.to(socketId).emit('verification_result', data.verifications[0].status);
+        io.to(socketId).emit('verification_result', data.verifications.identity_document.status);
```
Comment (dodgejoel-stripe) [inline server/server.js]: this is one of the bugs that was uncovered?

File: package.json [hunk 2]
```diff
@@ -23,6 +23,8 @@
     "node-fetch": "^2.6.0",
     "socket.io": "^2.3.0",
     "stripe": "^7.1.0",
+    "stylelint": "^13.2.1",
+    "stylelint-no-unsupported-browser-features": "^4.0.0",
```
Comment (dodgejoel-stripe) [inline package.json]: do the various es5ish changes flow from including this package?

File: client/js/next-step.js [hunk 3]
```diff
@@ -1,66 +1,27 @@
...
 
-const unhide = (element) => {
+var unhide = function(element) {
   if (element && element.classList.contains('hidden')) {
     element.classList.remove('hidden');
     element.classList.add('unhide');
   }
 }
 
-/*
```
Comment (dodgejoel-stripe) [inline client/js/next-step.js]: Is there not a progress bar at all or does it's progress get computed in some more natural way now?

Comment (bz-stripe) [reply client/js/next-step.js]: there's no progress bar anymore

File: server/server.js (hunk 1 above)
Comment (bz-stripe) [reply server/server.js]: yup

File: package.json (hunk 2 above)
Comment (bz-stripe) [reply package.json]: naw, I just added to inspect our CSS compatibility

I made the ES5 changes based on my internal understanding of browser support and some testing in IE11

but it's a good reminder to use `eslint-plugin-compat`

**Pull request merged**